wdfn-server/waterdata/templates/monitoring_location.html

```diff
@@ -12,12 +12,14 @@
             <meta property="og:url" content="{{ url_for('monitoring_location', site_no=stations[0].site_no, _external=True) }}" />
             <meta property="og:type" content="website" />
             <meta property="og:title" content="{{ page_title }}" />
-            <meta property="og:description" content="{{ components.Description(stations[0].site_no, location_with_values, parm_grp_summary) }}" />
+			<meta property="og:description" content="{{ components.Description(stations[0].site_no, location_with_values, parm_grp_summary) }}" />
```

Comment:
Some funky whitespace here. We use spaces to format and indent rather than tabs. I suspect that you are using tabs.